### DISCUSSION TITLE
Nach automatischem Update kein Web Oberfläche mehr

### DISCUSSION THREAD

**Discussion by Andy2020-DE** [OpenCCU/OpenCCU#2487] [🇩🇪 Fragen&Antworten]
### Describe the issue you are experiencing

Nachdem gestern das System ein automatisches Update gemacht hat, ist es mir nicht mehr möglich auf die Homematic Oberfläche zuzugreifen.

Die Login Maske wird angezeigt aber es Fehlen schon Texte.
Es wird >> ${lblEnterUserName} << und >> ${lblEnterPassWord} << angezeigt.

Ein Login ist möglich aber dann sehe ich nur das RaspberryMatic Logo (links oben) der rest ist leer.
NodeRed läuft noch und kann über die Weboberfläche auch bedient werden.

Am Raspberry 4B blink die LED rot und blau

### Describe the behavior you expected

Das Update sollte eingespielt werden und alles sollte so laufen wie vorher.

### Steps to reproduce the issue

1.
2.
3.
...

### What is the version this bug report is based on?

3.71.12-20231014

### Which base platform are you running?

rpi4 (RaspberryPi4)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
Mit PuTTY komme ich auf den Raspberry und kann mich dort als root anmelden. 
Sehe hier dann root@homematic-raspi:~#

Weis aber nicht was ich dann machen soll.
```

**Comment by blackbasket** [comment]
Das klingt sehr nach Cache leeren und ist eher kein Bug...

**Comment by Andy2020-DE** [comment]
Ich habe schon die Browser Opera, Firefox und Edge versucht.
Hier habe ich den Cache auch geleert aber ohne Erfolg.

**Comment by arne182** [comment]
Ich habe ein ähnliches Verhalten. Login kann ich einfüllen aber springt immer wieder zurück und lässt mich nicht rein

**Comment by arne182** [comment]
I am getting closer to figuring out what the issue is. It has something to do with my squid proxy. Looks like if I have it running direct it works fine but over the proxy it fails.

**Comment by Andy2020-DE** [comment]
Hallo Arne182, ich habe hier https://homematic-forum.de/forum/viewtopic.php?f=65&t=80264&start=10#p781381 auch schon ein Post abgesetzt, aber bislang noch keine Lösung erhalten. in dem Post, sind auch Bilder von der WebUi enthalten.

**Comment by steven83ger** [comment]
Hallo Arne und Andy, habt ihr das Problem lösen können? Bei mir ist seit vorgestern auch kein Login mehr möglich. (Mein Raspberrymatic läuft in HomeAssistant.) Backups zurückspielen brachte auch keine Besserung.

**Reply by arne182** [reply]
Ja, ohne Proxy. Geht dir Oberfläche

**Comment by Andy2020-DE** [comment]
Hallo steven83ger,

ich habe das System neu aufgesetzt und dann das Backup eingespielt, da ich sonst keine Lösung gefunden habe.

Das Problem hatte ich auch hier [https://github.com/jens-maus/RaspberryMatic/issues/2479#event-10732751708] geschildert, aber dort wurde es als "User Problem" deklariert.

**Comment by steven83ger** [comment]
Danke Andy für die Antwort. 
Also Du hast Backup erstellt, anschließend SD-Karte neu geflasht und dann das Backup zurückgespielt?

**Comment by Andy2020-DE** [comment]
Das Backup wird erstellt, bevor das System das Update durchführt.
Dieses Backup habe ich dann genommen und nach der Neuinstallation wieder eingespielt.

Neuinstallation = SD-Karte neu geflasht

**Comment by steven83ger** [comment]
Bei mir hats leider nicht geklappt. Raspberrymatic in Homeassistant gelöscht und neu installiert, Danach ging es.
Backup aufgespielt und es funktionierte nicht mehr. Auch mit alten Backups mit denen alles noch definitiv funktioniert hat klappen nicht.
Werde mir jetzt einen neuen Raspberry konfigurieren und dort Rasperrymatic aufspielen damit die Anwendung von Homeassistant getrennt ist.
Kann ich die Geräte einfach auf Werkseinstellung zurücksetzen (2x 4sec) und dann neu anlernen, oder gibt das Probleme?
Ablernen kann ich sie ja leider nicht wie zuvor im AccessPoint vom Homematic IP

**Comment by Andy2020-DE** [comment]
Hast du die Version von Raspberrymatic installiert von der das Backup war?
Ich meine ich habe erst die "alte" Version installiert, dann das Backup eingespielt und dann das Update auf die neue Raspberrymatic Version durchgeführt.

"Kann ich die Geräte einfach auf Werkseinstellung zurücksetzen (2x 4sec) und dann neu anlernen, oder gibt das Probleme?
Ablernen kann ich sie ja leider nicht wie zuvor im AccessPoint vom Homematic IP"

Da kann ich nicht zu sagen, das war bei mir nicht nötig.
